### ISSUE TITLE
using Fontconfig shrinks plot window relative to axes

### ISSUE THREAD

**Issue by nathanrboyer** [GiovineItalia/Gadfly.jl#1572]
This works fine.
```julia
julia> using Gadfly
julia> plot(Geom.line, x=1:3, y=1:3)
```
![image](https://user-images.githubusercontent.com/65452054/157712700-49b9c8dc-9353-42e9-880a-27d0afa64fc9.png)

This does not.
```julia
julia> using Gadfly, Cairo, Fontconfig
julia> plot(Geom.line, x=1:3, y=1:3)
```
![image](https://user-images.githubusercontent.com/65452054/157712817-20bc9c96-f364-4155-bd88-8feec7e74af8.png)

The problem becomes worse for more complex examples. The plot in this [discourse thread](https://discourse.julialang.org/t/plot-different-length-data-together-in-gadfly/77036/4?u=nathan_boyer) becomes unreadable. Using `draw(PNG("Test.png", 6inch, 4inch), p)` shrinks the plot even further.

**Comment by bjarthur** [comment]
by "this does not" work, are you referring to the rotated x-axis labels?  if so, try adding `Guide.xticks(orientation=:horizontal)`.

if you're referring to the size of the plot changing, as in the discourse thread, then i'm not sure that's a Gadfly problem per se as if i save the plot to a file and then it looks fine.  so pluto / vscode is probably wrapping the HTML in a weird way.  see https://github.com/GiovineItalia/Gadfly.jl/issues/1561

**Comment by nathanrboyer** [comment]
I am referring to the plot's size and distance from the axis labels. I see the same issue whether I save the plot to a file or view it as html.

**Comment by bjarthur** [comment]
i still can't reproduce.  me being on Gadfly master and julia 1.7.1 are the only differences.  i'm on mac osx. 
 tried saving to PDF, SVG, PNG, PS and they all look fine.

**Comment by nathanrboyer** [comment]
To reiterate, the problem only appears immediately after adding the Fontconfig package.

- If I only `using Gadfly`, then I can plot and save to SVG without any issues.
- `using Gadfly, Cairo` also behaves normally. 
- `using Gadfly, Cairo, Fontconfig` produces the weird plot dimensions above no matter where I render or how I save the plots.

I imagine I should raise an issue with Fontconfig.jl, but I don't know what to say is wrong or how it interfaces with Gadfly.

**Comment by bjarthur** [comment]
what OS are you using?

**Comment by nathanrboyer** [comment]
Windows 11

**Comment by montyvesselinov** [comment]
The same problem here!

**Comment by nathanrboyer** [comment]
Root cause was found in duplicate issue #1587, so I am closing this one. Temporary workaround is to pin a dependency of Cairo.jl to an older version.
```julia
]add Pango_jll
]pin Pango_jll@1.42.4
```

**Comment by montyvesselinov** [comment]
I confirm with additional testing that `Pango_jll@1.42.4` is needed. However, `Pango_jll@1.47.0` and `Pango_jll@1.50.0` fail!

```
]pin Pango_jll@1.42.4
```

**Comment by montyvesselinov** [comment]
I do not think this issue has been resolved.

**Comment by montyvesselinov** [comment]
@nathanrboyer have you found a solution to this problem? Other than using on old version of Pango_jll?

**Comment by nathanrboyer** [comment]
No, I stopped using Gadfly because of it. I only closed this issue because yours (#1587) is a duplicate, and there should not be two separate issues tracking the same problem. I haven't seen any indication that anyone has worked on this problem; presumably none of the current maintainers use Windows.
